src/ErrorPopup.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+
+interface Props {
+    error: Error|null;
+}
+
+const style: React.CSSProperties = {
+    zIndex: 2,
```

Comment:
these should just go into the stylesheet with a class

Reply:
done